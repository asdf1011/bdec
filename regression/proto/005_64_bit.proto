message fixed64types {
    required fixed64 a = 1;
    required sfixed64 b = 2;
    required double c = 3;
}
