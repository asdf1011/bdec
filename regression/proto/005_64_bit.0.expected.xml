<fixed64types>
    <a>18446744073709551615</a>
    <b>9223372036854775807</b>
    <c>3.14159265359</c>
</fixed64types>
